.builder/action/aws-c-http-test.py

```diff
@@ -0,0 +1,39 @@
+
+import Builder
+import sys
+import os
+import re
+
+
+class AWSCHttpTest(Builder.Action):
+
+    def _export_env_var(self, filename, env):
+        file = open(filename)
+        pattern = re.compile("(\w+) (\w+)=(.+)")
+        for i in file.readlines():
+            match = pattern.match(i)
+            if match != None:
+                env.shell.setenv(match.groups()[1], match.groups()[2])
+                print(match.groups()[1])
+
+    def run(self, env):
+        actions = []
+        if os.path.exists('/tmp/setup_proxy_test_env.sh'):
```

Comment:
how did this file get into `/tmp/`? does some previous step do this? please leave a comment